.gitignore

```diff
@@ -77,6 +77,7 @@ nbproject/private/
 
 # E2E and API Test Results
 test-results
+.auth/
```

Comment:
Which files are we are trying to ignore here? I couldn't find the auth folder on my local other than `plugins/woocommerce/templates/auth`.

Reply:
They get added to the e2e test directory I think as a result of logging in with specific storage states.

Reply:
Found it. Thank you!  🙂

<img width="540" alt="image" src="https://github.com/woocommerce/woocommerce/assets/11503784/c310b77e-9e24-466e-825e-20014f33d501">

Reply:
> Which files are we are trying to ignore here? I couldn't find the auth folder on my local other than plugins/woocommerce/templates/auth.

@tarunvijwani What @opr said in https://github.com/woocommerce/woocommerce/pull/43057#discussion_r1448915213. When running the test, these files get created for the storage states, but should not be pushed into the repo.